csharp/Platform.Data.Doublets/ILinksExtensions.cs

```diff
@@ -16,8 +16,24 @@
 
 namespace Platform.Data.Doublets
 {
+    ///<summary>
+    ///<para>.</para>
+    ///<para>.</para>
+    ///</summary>
     public static class ILinksExtensions
     {
+        ///<summary>
+        ///<para>Creates links with random values in the specified links storage.</para>
+        ///<para>Создаёт связи со случайными значениями в указанном хранилище связей.</para>
+        ///</summary>
+        ///<param name="links">
+        ///<para>The links storage.</para>
+        ///<para>Хранилище связей.</para>
+        ///</param>
+        ///<param name="amountOfCreations">
+        ///<para>Number of operations to create links.</para>
+        ///<para>Количество операция для создания связей.</para>
+        ///</param>
```

Comment:
Each `///<` should be replaced with `/// <`. To keep the same style of code comments in all project's repositories.